aws-services-prod.json

```diff
@@ -0,0 +1,24 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Resources": {
+    "S3Extractor": {
+      "Type": "AWS::IAM::User",
+      "Properties": {
+        "Path": "/",
+        "Policies": [
+          {
+            "PolicyName": "stsAssumeRole",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": {
+                "Effect": "Allow",
+                "Action": "sts:AssumeRole",
+                "Resource": "arn:aws:iam::*:role/keboola-s3-extractor"
```

Comment:
Nejsem si jistý jestli to až takhle omezit. Co když ze stejného AWS accountu budou tahat data do různých projektů?
Možná bych dal "arn:aws:iam::*:role/keboola-s3-extractor*". U jiných služeb je ale možné zadat jakékoliv jméno role, tam si ale nejsem jistej jak to udělat bezpečně, to by se muselo ověřit někde.

Reply:
v tomhle se tak úplně nevyznám, ale vycházel jsem z toho co si psal na slacku:
https://keboola.slack.com/archives/CQ1ASK06M/p1590661168015900?thread_ts=1590653013.011700&cid=CQ1ASK06M

Reply:
Takovouhle věc prosím příště dejte vědět dopředu aby se dalo dohromady zadání. ted' nebudu mít čas se tomu věnovat.

Reply:
@Halama měl by si chvíli v příštím sprintu se na to podívat? abychom to mohli uzavřít...